### ISSUE TITLE
Proposed stun balancing.

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#8247]
A continuation of #8220 because the scope has changed slightly and some more detail should be added. Again, this is a step in the right direction, attempting for a less controversial change rather than a complete rehaul of everything stuns are which I do not have the time, ability, nor energy to do.

The basic premise is that when you are hit by something that should stun you for the first time, you are merely "knocked down". 

**When knocked down:**
- You cannot move
- You can still use weapons or items 
- If you're hit again by a stun you drop your weapons and are "properly" stunned, unable to use objects etc. - normal stun behaviour.
- You will be knocked down for as long as you would normally be stunned assuming no further stuns appear

As for RNG, I'm against it in general, so I feel like there's no need for it here. If you can give me an argument that RNG should be implemented, please do so.

**Comment by SonixApache** [comment]
being able to hold onto items while down will result in hilarious and awful issues, mark my words

**Comment by ghost** [comment]
Like what, @SonixApache ?

**Comment by SonixApache** [comment]
first thing that comes to mind is being able to perform surgery on yourself

**Comment by 9600bauds** [comment]
I'm not sure. This would affect ALL of the things that can currently stun, and it wouldn't affect shit such as chloral etc. Besides, you'd also take a fat dump over Sec because you're taking power away from their means of apprehending criminals.
I feel like a better way to approach this would be look at which stuns cause people to complain. I think the great majority of complaints about stuns revolve around eguns and ghetto stuns.

**Comment by ghost** [comment]
@9600bauds That's more of an issue with the power of chloral rather than the weakness of stuns in comparison, in my opinion.

**Comment by 9600bauds** [comment]
I just said a great deal of complaints revolve around eguns and ghetto stuns tho

**Comment by ghost** [comment]
There is literally nothing wrong with the current stun system.
There never has been.

The all-or-nothing nature of the current system rewards proper play and punishes sloppiness.

Also, all this does is ruins melee stuns and double gun energy usage.

**Comment by 9600bauds** [comment]
I TOLD YOU YOU'D WAKE UP SARCA

**Comment by ghost** [comment]
@9600bauds And I'd argue it actually is nowhere near a nerf to security - Indeed, in reality, it's a zero-sum game. It makes it equally harder for an antagonist to stun security as it does security to stun an antagonist.

As for ghetto stuns and eguns, there is a simple fix: If you've an example of a weapon that should deal both stunning as well as knockdown damage then there's no reason why it can't do that.

@ContententiousContent 

> There is literally nothing wrong with the current stun system.

Stopped reading there.

**Comment by Zth--** [comment]
Ghetto stuns should nerfed to hell. The weapon factory (aka rnd) should be nerfed too.

**Comment by AutismSpider** [comment]
so you cant move and you don't drop shit means force to lasers people to death if they have a gun hell or if they have strong melee weapon for matter of fact you cant go near them with out them hitting you

**Comment by FatBaldBastard** [comment]
There's no point in nerfing (Because this is what it is, not rebalance) stuns because they're the entire backbone of SS13 combat. Without it, it'll become the RNG fest that you dislike as people resort to cause bleeding and knockdown as next best thing.

Due to that, Security's job becomes much harder as reliable ways to take opponent down non-lethaly diminish greatly, forcing them to resort to lethal means to subdue opponents reliably. For example, stun baton will become utterly useless, for in CQC engagement you can be easily disarmed, injected with para-pen or chloral or just knocked prone from a well-placed punch before you can deliver the second incapacitating strike.

Cyborgs will be buffed as well, for what is stun to a laser/taser shitting death machine? It's already a bunch of boring crap to fight them with one-hit-stun flashes and ion rifles, imagine if they get a free pass?

To summarize, this will cause overall spike of lethal takedowns and discourage players from playing security either, as you have to use more charges of your ranged weaponry, forget about CQC weaponry and hope that the target will not whip out a laser cannon to burn your face off before you manage to land a second hit. Secborgs will become even more robust and unfun as the only reliable way that remains to combat them is elusive ion rifles. Ghetto stuns via rods and scrubgloves will be also less viable, making it even less of a match against ranged weaponry.

Everyone will suffer, but non-sec non-command non-guntotting cargo will suffer the most.

**I AM AGAINST THIS CHANGE**

**Comment by 9600bauds** [comment]
> And I'd argue it actually is nowhere near a nerf to security - Indeed, in reality, it's a zero-sum game. It makes it equally harder for an antagonist to stun security as it does security to stun an antagonist.

You're getting kind of tunnel-visioned here, the antagonist has absolutely no qualms about killing Sec (and if stuns are nerfed he might just open up with lethals), while the officer might not have lethals in code green.

**Comment by AutismSpider** [comment]
nothing stopping an tag form going leathals form the start with this if he does get drop he can keep fighting back with his ebow and esword if any one gets near him or her

**Comment by ghost** [comment]
@9600bauds The current meta is that you open with a stun and then lethal. This probably won't change that, it's a minor change that requires double-tapping rather than a single tap.

**Comment by ghost** [comment]
@AutismSpider They can just hit him twice to disarm him.

**Comment by FatBaldBastard** [comment]
This will require major changes to rules and sec SOP. This will also result in a buff of such thing as eswords, shields and protective armor, as even if first hit isn't tanked, the second hit might be.

Guns already have barely any charges, forcing you to double-tap will make security less of a force as it cannot tackle crowds anymore. Awful in the current low-sec-pop times.

**Comment by FatBaldBastard** [comment]
And what do you say about such things as Parapens, Magic Missile and Paralysis sting?

If they're still one hit stun, the station stands almost no chance. If they are not, the usage or timer limit will make them very much useless, nerfing antags to hell.

**Overall, fuck off with this uncalled change. Just because 2hotdogs whined about it in the thread doesn't mean we want it.**

**Comment by 9600bauds** [comment]
Nice bogeyman

**Comment by ghost** [comment]
@FatBaldBastard I'm saddened that you're unable to debate this in a calm, collected manner without resorting to bringing up the bogeyman of the week or claiming in any way you represent the majority.

**Comment by FatBaldBastard** [comment]
Don't attack my tone, these are valid concerns that you must address.

**Comment by ghost** [comment]
And they're concerns that can be talked about. A Possible option is that those types of stuns cannot be used when knocked down, and hence there is no change from the current system where those stuns are concerned.

**Comment by FatBaldBastard** [comment]
If they still can be used from normal state, it doesn't matter.

Ling, Traitor or Wizard has to gut you once, while you have to struggle to dodge their attacks while trying to land TWO hits.

Or ling, traitor or wizard must struggle with TWO parapens or timers to succeed. Bleh.

**Comment by ghost** [comment]
> I'm saddened that you're unable to debate this in a calm, collected manner without resorting to bringing up the bogeyman of the week or claiming in any way you represent the majority.

Brah

> Stopped reading there.

This change ruins the entire flavour of SS13 combat, and needlessly increases difficulty for non-antag crew to ever survive an adverse situation. Your average assistant with a fire extinguisher now cannot even powergame up some ghetto stuns, he is literally reduced for fire ext or floor tiles. In either case, he has no way of taking down an opponent without getting hurt, and two assistants fighting will frequently result in the death or equivalent (painlock for 80% of the game, lethal internal bleeding, etc.).
This is because melee stuns are virtually removed by this change, as anyone with a more reliable stunning method (e.g. chemists, antagonists, security, etc.) are almost certainly able to take you down before you can double tap with your extremely limited charge.

This also turns the station into even more of a 'call out on radio for lynchmob' oriented system, because it takes longer to get the initial stun condition on before you can start to disable their ability to take lateral action against you.

This idea has been had before and it was always bad. Direct combat (open and declared) makes up maybe 20% of murders, if that, which environment and stealth kills are more frequent. This change skews direct combat and especially stealth hugely in the favor of those who already hold the advantage (antag/sec/chem vs. e.g. assistants) and does nothing to address the fact that Eguns and family are a huge step above other stun options because it **buffs Eguns relative to stunbatons/etc.**

**Comment by FatBaldBastard** [comment]
Also, take notice of high-damage lethal weaponry. Stuff like shotgun deals enormous amount of damage in one hit, often breaking bones, causing bloodloss and it might even tear your limb off. At this point it's even meta to use shotguns to combat things that are less affected by stuns such as wizards. This weapon is much more reliable and useful as it takes person down, often permanently, in one good hit, something that stun weaponry does not do in this situation. It's now a prefered self-defense method. What're you gonna do when people will say "Shotgunning him was much safer for me and others than stunning him and letting him fire shots at me or detonate bombs" and will be completely in the right? Still ban them despite ruining the only way of resolving situations without death? Revise murder rules?

**Comment by 9600bauds** [comment]
If there's an easy way to improve the combat system relative to stuns, this is just not it.

**Comment by AutismSpider** [comment]
hell the use of shots guns would go up move this makes stuns out right useless antags would use the 357 more that ebow is dog shit now and you can two shot people with the 357 any way

**Comment by ghost** [comment]
@9600bauds Apparently, this is quite similar to the system goon has, and honestly, unless you've a better idea, I think this is the way forward.

**Comment by FatBaldBastard** [comment]
The idea is to not touch the stun system. For some reason you do not accept any solution but to change it, when the change is most unwelcome and will create so much work for supposedly something that "won't change the meta".

There's a page full of reasons why touching stuns will ruin a lot of things, while benefits are barely there. So why is it worth it to change it?

Overall, I noticed that you list exactly zero ways it'll benefit the server, community and gameplay. Your posts are just "Let's change it to this", no rhyme or reason. So do tell us why exactly you want that in.

**Comment by MusgardTiger** [comment]
You guys are arguing that you have no counter against lings and wizards. People already don't use stuns to take care of wizards or lings. You don't get close to ling, you laser it until it dies and then you cremate/gib it and it is not like people get a free stun invulnerability card, they'll get slowed down. Landing the second shot wouldn't be an issue for whomever using the stun gun. Fighting with stuns is a routine, it is not rewarding in a sense of gameplay. It is rewarding in a sense of a chore. You yakity sax and flail around like crazy to dodge stuns and you hope you can land one on the other guy while moving around like a hobo on meth. This is a routine, there is no flavor to it. 

I am not saying we should just push this in, I am saying "NNNNNNNNNNNNO WE SHOULDN'T EVEN DISCUSS IT" is pretty dumb. We should find a way to make this game feel like a game instead of a set of fighting chores you have to get done to earn your greentext/or get your valids.

**Comment by FatBaldBastard** [comment]
The way change is worded, the yaketi-sax is still there, except first stun still lets the guy to fire a victorious 357 shot into your gun hand. It's even more of a chore. And how do you expect to laser a ling that's yaketi-saxing away? It's much easier to waste him when he's on the floor doing nothing rather than shooting back, tossing grenades or just moves a locker between himself and you.

**Comment by LordMyroc** [comment]
Stuns could use a rework.

This is not it.

**Comment by AutismSpider** [comment]
@DrCelt Stun drop to floor+ drop items with the goon one on stop of the chems they can take to have stun res any way

**Comment by MusgardTiger** [comment]
I am not a coder, call me on my dumb shit if it is undoable. First shots should cause negative effects like wonky aim (firing somewhere which you are not aiming at) impaired speech and sight and slow movement and a chance to drop whatever you are holding for like six seconds and you gradually get better like in 4 seconds and second shot knocks you on your ass. If you can land the first shot on the ling who is yakety saxing you are impairing him greatly you can easily switch to lethals if you deem it necessary in that disabling time and recovery. You are still risking everything when you get in a fight with stuns but it is not one shot goodbye hit, you'll be encouraged to take cover and find a weakness in your opponents moves because you'll know that there is a chance that you might survive/win this encounter but you are still risking yourself greatly. There is a hope factor.

Now, when you are in a stun fight with another player you definitely know you are fucked if he takes a lucky shot, there is no hope factor. You either do the meth dance and empty your taser in his general direction or get stunned because he emptied his taser in your general direction first.

**Comment by Shadowmech88** [comment]
Frankly I don't see how the current proposed solution would change anything. If the first shot stops them from moving, then it'd be just as easy to double-tap someone as it would to normally stun them.

**Comment by AutismSpider** [comment]
laying down on the floor makes laser shots etc pass right over you some times.

**Comment by ghost** [comment]
This is the problem with changing stuns. You have people like @FatBaldBastard or @ContententiousContent who claim that stuns don't need changed at all, and then you have people who say the change either doesn't go far enough or that it won't make a difference.

I will ask pomf to make a poll with the title  "Does the current stun system require tweaking?", which I am almost certain will return a yes vote. This is just an option of many, and I would encourage anyone in the case of it succeeding to propose their own tweaks, and a vote for stun system requiring tweaking is not a vote for these changes.

**Comment by FatBaldBastard** [comment]
You're yet to make a case why they should be changed, while we made imho strong cases against the change. Before this thing goes anywhere beyond this proposal, please propose this properly and say why exactly the server will benefit from these changes.

**Comment by ghost** [comment]
@FatBaldBastard I can make a more effective case for this being the change I think is best once it's agreed that a change is necessary in the first place. As for stuns needing changed in general, that's a topic for a different discussion.

**Comment by FatBaldBastard** [comment]
It is very clearly not agreed that change is neccesary, especially in this aspect, for people oppose it right here in this issue.

**Comment by ghost** [comment]
@FatBaldBastard Well that's why I'm asking pomf to make a poll.

**Comment by Shadowmech88** [comment]
@DrCelt 
Well you wouldn't bother asking Pomf for a poll if you yourself didn't think stuns needed to be changed.

So why do you, personally, think stuns need to be changed?

**Comment by ghost** [comment]
@Shadowmech88 
I would prefer to articulate this in a separate issue but to summarise:
- The current meta disincentives all but the most powerful of lethal weapons. (Even pulse rifles, the most powerful lasers in the game, benefit from the stun-then-attack)
- Landing a stun ends combat. There is no comeback from being stunned 99% of the time.
- In a winner-takes-all scenario. Stuns are, ironically, the most lethal item in the game. 
- Combat being BZZT and it's all over is not entertaining to participate in or to watch
- Stun landing between two players often becomes more about luck and who can greytide waltz and spray the most, for the longest and fastest than whether or not you are capable of firing a good shot.

**Comment by ghost** [comment]
> The current meta disincentives all but the most powerful of lethal weapons. (Even pulse rifles, the most powerful lasers in the game, benefit from the stun-then-attack)

That's because any alternative results in you taking a large amount of damage or dying because they are able to respond with an equal force.
Also it is literally only a meta issue, in the sense that most good players (i.e. the minority) still know you can laser someone into crit before they react or floor tile slow or evasive targets with ease and so happily use those options.

> Landing a stun ends combat. There is no comeback from being stunned 99% of the time.

This is good (in my opinion). This is purely a question of taste, but I prefer this system because it rewards players who don't make mistakes and punishes you instantly for mistakes (getting stunned in the first place is unquestionably and objectively the result of a mistake on your part).

> In a winner-takes-all scenario. Stuns are, ironically, the most lethal item in the game.

So?

> Combat being BZZT and it's all over is not entertaining to participate in or to watch

Subjective. 
Also this really doesn't apply to combat, only stealth kills.
Defense outstrips offense in growth with skill, good players take longer to stun each other and combat spills out onto a larger area and generally becomes more interesting with competency, interesting set piece combats which are fun to watch or participate in do happen but are rare.
Actual 1hitstunfightover usually only happens when one party is taken by surprise, which hardly qualifies as combat.

> Stun landing between two players often becomes more about luck and who can greytide waltz and spray the most, for the longest and fastest than whether or not you are capable of firing a good shot.

There's no such thing as luck, just shitters trying to shift blame away from their incompetence.

**Comment by ghost** [comment]
> There's no such thing as luck, just shitters trying to shift blame away from their incompetence.

hahahahahahaha

**Comment by Imagebard** [comment]
>There's no such thing as luck, just shitters trying to shift blame away from their incompetence.

Are you kidding me? SS13 combat is fucking abysmal.
I can't fucking wait for the Doom engine remake.

**Comment by Shadowmech88** [comment]
Actually melee combat is more luck-based than ranged combat is. RNG can cause you to miss while swinging an axe at your target, but if your aim is on point a taser bolt is never not going to stun your target.

**Comment by AutismSpider** [comment]
Only thing that comes to luck is ebin disarm push/trying fight people with melee weapons that is pure  rng.

**Comment by ghost** [comment]
I don't think we can ever convince you, and we could certainly argue for days on this topic without either of us budging,  @FatBaldBastard / @ContententiousContent but can we perhaps agree to leave this for the community to decide in a server poll?

**Comment by Imagebard** [comment]
>but if your aim is on point a taser bolt is never not going to stun your target.

There's a chance of missing ranged attacks, you know. Doubles if you're clumsy(Ex: The clown).

**Comment by ghost** [comment]
> Are you kidding me? SS13 combat is fucking abysmal.

There's a heavy RNG presence, but I mean that things like e.g. tasers or stunbatons with no RNG involved have literally 0% to do with luck.

Melee combat with low tier weapons is purely luck based though. I've taken out people with eswords with fireextinguishers just because of a good roll, and that's the only stun system I would say might be in need of a change.
That said, you can bypass that stun system by using higher tier weapons.

Actually, the whole system is progressively less RNG the higher tier weapons you use. Atmos & bombs have 0% rng, Eguns have a small % RNG (now, this should probably be reverted) and relatively small microprediction requirement, melee stuns have high microprediction environment, while plain melee is 100% RNG for stuns, and punches are 100% RNG for stuns AND damage.

> I don't think we can ever convince you, and we could certainly argue for days on this topic without either of us budging, @FatBaldBastard / @ContententiousContent but can we perhaps agree to leave this for the community to decide in a server poll?

No, fuck you.
Obviously if there is massive community support then I have no illusion that this would be changed, but running polls until you find one with a bias in your favor so you can force in terrible changes for literally no reason has nothing to do with well reasoned arguments.
People who don't understand how bad this would be and would regret putting it in and shitters who basically don't get involved in combat but just want to force shitty changes for the sake of change are going to support you in voice and votes, but don't actually give a reason to make the change.

**Comment by ghost** [comment]
Also the greytide waltz can still be faster than an on point aim.
@ContententiousContent That's a little rude. If your point is defensible, then convince the server. I will attempt to do the same. I don't think we should avoid changing things due to a silent minority, which as far as I am concerned, you are. I gave reasons to make changes above and while, yes, they're subjective, so are your refutations. Therefore the best way to resolve this is to find out who's subjective point of view is more widely held.

**Comment by ghost** [comment]
Ability to convince the server has nothing to do with the validity of your points or even what people would enjoy more, just your ability to sell the idea to them.
We also have a strong 'change for the sake of change' culture among really all of ss13  in the vocal minorities which biases the userbase by propaganda.

Literally when it comes down to it if you want to treat popular vote as a valid measure of whether or not something is a good idea then you're saying that the opinions of astroturfers are more valid than anyone else's.

> That's a little rude

Jesus Christ man lighten up

**Comment by ghost** [comment]
I don't accept the argument that people aren't smart enough to decide what happens on their own server, @ContententiousContent

**Comment by ghost** [comment]
> I don't accept

Coolio

**Comment by Imagebard** [comment]
@ContententiousContent 
No, I think you misunderstand.

SS13 combat is abysmal because BYOND is abysmal.
The only good Player versus Player interactions we have is when we stop using weapons(That means no eguns, no tasers, no e-swords) and start using indirect means such as when you shock doors you know your target is going to go through, when you hide bombs/grenades, and otherwise whenever we think outside of the box. Actual melee/ranged combat is Intrinsically shit and will continue to be as long as we are on BYOND.

**Comment by ihadtoregisterforthis** [comment]
The vote must have every option to be worth anything.

**Comment by ghost** [comment]
@ihadtoregisterforthis Every option?

**Comment by ghost** [comment]
@Imagebard 
Actual melee/ranged combat might be intrinsically shit, but as you say that's unavoidable on byond so it's symptomatic of an unrelated issue.

It's also among the least effective methods of robusting people, so it doesn't have that much impact on the game itself anyway.

**Comment by FatBaldBastard** [comment]
@DrCelt 
There are countless options on how to go about this and distilling them into PRESS BUTTON TO OPINION will take a long time and will provide with at least ten options, ranging from severity of change  or lack of it.

This can't be decided by a simple vote, people must come here and place their own, described opinions.

**Comment by ghost** [comment]
@FatBaldBastard We could always have Pomf set up those fancy input box votes where you have a textbox as your "vote"

**Comment by ihadtoregisterforthis** [comment]
On the poll, if it's between a variety of options, make sure no option combination is left out, except for those that directly rule out branches of options such as "Do not change the stun system"

**Comment by FatBaldBastard** [comment]
@DrCelt 
Except where will we examine these votes and who will we hold accountable for that? This has no reason to be anonymous, because it requires a seasoned player to judge on this, not someone who gets robusted every day and begs for the difficulty to be turned down.

as for your points.

> The current meta disincentives all but the most powerful of lethal weapons. (Even pulse rifles, the most powerful lasers in the game, benefit from the stun-then-attack)

Shocker, you fight armed people with superior weapons and want them to deal minimal damage to you. You bring in these changes and meta will adapt and you'll see people putting shotgun bullets into other's eyes for quick takedown. More so, tasers are also admin-safe option. With length of bans and history staying with you for a long time, people resort to tasers because if there's no intention to kill, it's the most reliable, safe and ROBUST way of dealing with the enemy.

> Landing a stun ends combat. There is no comeback from being stunned 99% of the time.
> Depends on the skill of a player, I often stand up from scrubgloving because people just aren't fast.

Either way, with weapon curve going from barely lethal extinguishers to decapitating axes and barely damaging stun revolver to whopping 80 damage 357 or fast-firing SMG, you won't be hurt or just won't come back either. It's not the problem of stuns, it's the problem of overall deadliness of combat when you encounter an equiped and skilled player, as well as half of weapons being useless while other being extremely effective.

> In a winner-takes-all scenario. Stuns are, ironically, the most lethal item in the game.

Arguable, considering that there's a lot of items that very much counter ranged stuns (esword, eshield, abalative armor, normal shield might help as well, reactive armor), while lethal weapons are not countered as much as it's effectiveness is reduced. Abalative armor can fire shots back at you, but bulletproof vest doesn't do that. So if I fight against egun-totting bulletproof armor guy while I have 357, abalative armor and esword, I have higher chance of winning just by having RNG be there as failsafe if my skills are not enough. And If I'm ranged against melee stuns, they just can't reach me as I fill them up with lead or laser bolts.

Skill decides.

> Combat being BZZT and it's all over is not entertaining to participate in or to watch

Not for you maybe. High risk high reward combat of SS13 is the best adrenaline rush I've had in videogames since forever and skilled players can really make it look fine (And I dont mean just spamming direction keys, I mean relocating to advantegous positions, retreating to friendly forces, fleeing, advancing under cover of flash and smoke, etc). If you expected something more from a 2D Atmospherics simulator, well I'm sorry to disappoint.

> Stun landing between two players often becomes more about luck and who can greytide waltz and spray the most, for the longest and fastest than whether or not you are capable of firing a good shot.

Well duh, dodging is important, who wants to be hit? As you said, the meta won't change and grey waltz will remain, it's just you'll waste another shot to stop the opponent for good. It's useless and a waste of time, fixing nothing but making combat more annoying.

I think my word carries special meaning, since no matter what you say, I spent the last 6 years using this system to fight people, and I return again and again to it and it's still fresh and fun. I've had great moments and battles, lots of victories and defeats, lots of close calls that you seem to think are non-existant. I suggest you take a break from coding and admining and spend at least a week playing as Security Officer and fight using this system. I am sure you'll realize that you're mistaken.

**Comment by Blithering** [comment]
Might as well put it in this one too. Rather than a blanket nerf to stuns, why not put this change into a chemical and make it available in the form of implants or stimpacks for traitors and ERT respectively? So if your average greytide is running around being a dick, one shot will still down him, whereas someone who's planned ahead can take a shot or two before getting taken down (and bonus points if getting shot while under it's influences causes the player to yell RIP AND TEAR etc.)

**Comment by FatBaldBastard** [comment]
> Might as well put it in this one too. Rather than a blanket nerf to stuns, why not put this change into a chemical and make it available in the form of implants or stimpacks for traitors and ERT respectively? So if your average greytide is running around being a dick, one shot will still down him, whereas someone who's planned ahead can take a shot or two before getting taken down (and bonus points if getting shot while under it's influences causes the player to yell RIP AND TEAR etc.)

I guess it's a more adequate solution, but it has to be balanced proper since it's hulk with one major bonus and no downsides. Gotta be something like Creatine where you gotta balance dosage to not die and still remain stun-resistant.

And it doesn't hurt the rules or force escalation, since if not for the drug we could've taken him in non-lethally, but since it fails he gets the hulk treatment. It's the blanket stun that fucks it up since you have to treat everyone like hulk to reliably survive.

**Comment by Blithering** [comment]
I'd say have the downside be something like it speeding up the hunger gauge a lot, with a hulk end style blackout when it wears off. The main difference between hulk and it would be no additional strength, and multiple stun shots still being able to take them down.

**Comment by Shadowmech88** [comment]
One of the main drawbacks to hulk though is that you can't use guns. These stimpaks would confer stun resistance while still allowing you to use stun weaponry yourself, which is a big advantage.

**Comment by Imagebard** [comment]
Hulk gives you stun *immunity*, stimpaks would confer stun *resistance*.

**Comment by Shadowmech88** [comment]
I know. What I'm saying is that stun resistance+guns would be more valuable in spess combat than stun immunity+noguns.

**Comment by Blithering** [comment]
You're forgetting hulkism also confers the ability to punch through walls and faces, as well as the ability to fight on while in crit, and if that isn't enough of an upside also consider that one of the proposed effects of being shot by a stun weapon that doesn't instantly down would be random discharging of firearms.

**Comment by Shadowmech88** [comment]
I'm referring purely to the proposed method of functioning for stimpaks.

Additionally, a hulk can only attack one person at a time due to being melee-restricted. Two well-coordinated people armed with lasers can take down a hulk if it's not assisted by other chems.
Meanwhile someone on a stimpak, who would have stun resistance, could take on three people and use a taser/laser combination to kill all three, since his stun resistance means that it won't matter if he gets hit once or twice.

**Comment by Imagebard** [comment]
>or twice

If he gets hit twice means he won't be able to shot.
In fact I don't know why stimpacks are such problem. It would only be useful if 
A. You dodged everything BUT the last shot someone made
B. The person attacking you didn't know you were on stimpaks

**Comment by 9600bauds** [comment]
I'd just like to interject for a moment and say
>powergaming stun-resistant chems AS A SOLUTION
Jesus

**Comment by AutismSpider** [comment]
@9600bauds even goon has stun resistant chems  but most of them are antag only  ones that  player made have massive down sides to point were they kill you.

**Comment by kilkun** [comment]
@autismspider only if you dont know how to do proper chem on goon.

The chemicompiler has made it ezaf to micro a super chem that confers meth + secret nostun + +1 all heal per tick.

Stun resistance via chems should be done via natural secreting adrenaline, which is also available in implant form for special roles, like I originally suggested.

**Comment by AutismSpider** [comment]
implant should be for Nuke Ops and Sec then unless the implant going bug out the ops in some way

**Comment by ghost** [comment]
In order to start balancing stuns, you must first resort the underlying problems. Mainly movement speed being too fast for a more controlled combat environment to emerge.

**Comment by FatBaldBastard** [comment]
What the fuck is even powergaming, how is it even an argument. Knowing how and where to acquire stuff you need isn't powergaming, the true meaning of this word doesn't even apply to the level of RP that's present on /vg/station. Stop using buzzwords.

@TermedClepe

Making combat a controlled environment turns the act into something that isn't combat. Combat can't be a controlled environment by definition.

**Comment by ghost** [comment]
Is there a way to filter posts on github?

**Comment by Imagebard** [comment]
@FatBaldBastard 
How the fuck did you become an admin

**Comment by FudgePucker** [comment]
A possible solution to make securitiy's life easier might be to make stun batons 100% chance while ghetto prods are lower and stun gloves slightly lower than that. Make new addrinialin implants given round start to security and captain and a lockbox of them in armory, NOT ORDERABLE FROM CARGO, that gives resistance to full knockdowns.

Now security isn't nerfed by this change, they are buffed. That greyshit can't shove you down quite as easily, and you can still baton them.

**Comment by ghost** [comment]
@FudgePucker Well that could certainly work, and would definitely work alongside the proposed changes, but again I'm not sure about introducing more RNG

**Comment by FatBaldBastard** [comment]
Issue is, that gives combat-focused roles even more of an advantage over other crew members and sec-less rounds will feel even more of a drag if the opponent is new cop or a traitor with this implant.

**Comment by IratePirate** [comment]
Fuck it, anything is better than the current stun system the game has had for the last decade.

**Comment by Intigracy** [comment]
But the current stun system the game has had for the last "decade" works.

Other servers have a two hit stun system. It sucks ass.

**Comment by Brolaire** [comment]
The current one is fine imo. Just give different outfits and armors different %-stunres
